.github/workflows/ci.yml

```diff
@@ -3,6 +3,9 @@ name: Run CI tests
 on:
   push:
     branches: [dev]
+    paths:
```

Comment:
(Ah, hadn't realized this PR is still draft, sorry if I jumped the gun)

Reply:
All good, its a good catch `test/**` needs to be included as well. I set it to draft because I quickly made these changes and had to go look at something else, so was planning on coming back to it.